kamu-core/src/infra/repos/object_repository_in_memory.rs

```diff
@@ -59,6 +72,10 @@ impl ObjectRepository for ObjectRepositoryInMemory {
         unimplemented!()
     }
 
+    async fn get_download_url(&self, _prefix_url: &Url, _hash: &Multihash) -> Result<(Url, Option<DateTime<Utc>>), GetError> {
+        unimplemented!()
```

Comment:
I used `unimplemented!()` above for methods that can be implemented but there was simply to use case for doing it back then. In this case `panic!()` is probably more appropriate as it cannot ever be implemented.

Reply:
Makes sense